src/demos/input-time-picker.html

```diff
@@ -81,6 +81,35 @@
           <calcite-input-time-picker scale="l" step="1" value="23:59"></calcite-input-time-picker>
         </calcite-label>
 
+        <calcite-label scale="l">Placement: bottom, timePickerScale: m</calcite-label>
```

Comment:
Instead of updating the demo page, can you add storybook entry? Then our screenshot testing will cover this. We can provide guidance on this if needed.